Tools/CISupport/ews-build/config.json

```diff
@@ -287,6 +293,12 @@
       "triggered_by": ["macos-ventura-release-build-ews"],
       "workernames": ["ews181", "ews182"]
     },
+    {
+      "name": "macOS-Safer-CPP-Checks-EWS", "shortname": "mac-safer-cpp", "icon": "buildAndTest",
```

Comment:
Please verify whether `testOnly` icon would be more appropriate for this queue.

Reply:
Upon discussion with @ryanhaddad, `buildOnly` is actually the best option, since it's a custom build process rather than a test suite that's being run.